lib/actions/deploy.coffee

```diff
@@ -70,7 +84,7 @@ uploadToPromise = (request, size) ->
 			if obj.type?
 				switch obj.type
 					when 'error' then reject(new Error("Remote error: #{obj.error}"))
-					when 'success' then resolve(obj.image)
+					when 'success' then resolve([obj.image, obj.buildId])
```

Comment:
I'm a big fan of objects for things like this (as mentioned in my other comment)